src/lib/common/statistics.cpp

```diff
@@ -211,5 +214,8 @@ void statisticsUpdate(RequestType request, Format inFormat)
   case EntitiesResponse:                                 ++noOfEntitiesResponses; break;
   case EntryPointsRequest:                               ++noOfEntryPointsRequests; break;
   case EntryPointsResponse:                              ++noOfEntryPointsResponses; break;
+  case EntityRequest:                                    ++noOfEntityRequests; break;
+  case EntityResponse:                                   ++noOfEntityResponses; break;
+
```

Comment:
The counters should be added to the output of 'statisticsTreat' also (in serviceRoutines)